src/components/LandingPage/index.tsx

```diff
@@ -9,11 +10,13 @@ import BlankCard from "../BlankCard";
 import Footer from "../Footer";
 import { TUTORIAL_PATHNAME } from "../../routes";
 import { CYTOSIM_URL, READDY_URL } from "../../constants";
+const markdown = require("../../../ACKNOWLEDGMENTS.md");
```

Comment:
To double check, acknowledgments.md will be found at the same relative path in the deployed website?

Reply:
Good question... I think that the webpack loader should import the md file as a module and package it all up in the dist folder, so I think we'll be good, right @meganrm ?

Reply:
yeah